src/js/components/Notification/Notification.js

```diff
@@ -0,0 +1,30 @@
+import React, { forwardRef } from 'react';
+
+import { Layer } from '../Layer';
+import { Box } from '../Box';
+
+const Notification = forwardRef(({ message, onClose, ...rest }, ref) => (
+  <>
+    <Layer
+      position="center"
+      animation="slide"
+      modal={false}
+      ref={ref}
+      onEsc={onClose}
+      onClickOutside={onClose}
```

Comment:
I'm not sure we want `onClickOutside`. If the Notification renders just as the user is about to click something on the screen, the Notification will immediately be removed.